### PR TITLE
Correct some typos

### PR DESCRIPTION
**Pull request by goggle**
Corrects some typos in the repository.

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ reStructuredText <http://sphinx.pocoo.org/rest.html>`__.
 The documentation follows the
 `NumPy Docstring Standard <https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt>`__,
 which are parsed using the
-`napolean extension for sphinx <http://www.sphinx-doc.org/en/1.5.1/ext/napoleon.html>`.
+`napoleaon extension for sphinx <http://www.sphinx-doc.org/en/1.5.1/ext/napoleon.html>`.
```
Comment (tompollard) [inline docs/contributing.rst:49]: The `a` sneaked into your fix! Please could you remove it (`napoleaon` -> `napoleon`)?

**Review by tompollard — APPROVED** [review]
Looks good to me, thanks! Please could you correct the `napoleaon` before we merge?

File: docs/contributing.rst (hunk 1 above)
Comment (goggle) [reply docs/contributing.rst]: I'm sorry, it has been corrected now.

**Comment by tompollard** [comment]
Thanks @goggle!

**Pull request merged**